janus_core/log.py

```diff
@@ -59,19 +77,10 @@ def format(self, record: logging.LogRecord) -> str:
         record.msg = record.msg.replace('"', "'")
 
         # Convert new lines into yaml list
-        if len(record.msg.split("\n")) > 1:
-            msg = record.msg
-            record.msg = "\n"
-            for line in msg.split("\n"):
-                # Exclude empty lines
-                if line.strip():
-                    record.msg += f'    - "{line.strip()}"\n'
-            # Remove final newline (single character)
-            record.msg = record.msg[:-1]
-        else:
-            # Wrap line in quotes here rather than in FORMAT due to list
-            record.msg = f'"{record.msg}"'
-
+        msg = record.msg
+        record.msg = "\n" + "\n".join(
+            [f'    - "{line.strip()}"' for line in msg.split("\n") if line.strip()],
```

Comment:
Works with a generator, no need for extra brackets
```suggestion
            f'    - "{line.strip()}"' for line in msg.split("\n") if line.strip()
```